geoapi/utils/users.py

```diff
@@ -1,5 +1,18 @@
 class AnonymousUser:
-    username = "Guest"
+    """
+    Represents an anonymous user.
+
+    Attributes:
+        username (str): A unique username for the anonymous user, prefixed by 'Guest_'.
+                       If no `guest_unique_id` is provided, username defaults to 'Guest_Unknown'.
```

Comment:
Under what conditions would this occur? Regardless of user vs guest state, every connected client session should have a UID of some kind, right?

Reply:
We are generating the uuid on the client side (i.e. hazmapper/taggit) and then adding to all requests (we put it in 'X-Guest-UUID' in the header).  So if someone else was making requests outside of taggit/hazmapper then there would be no `X-Guest-UUID` in the header. So interacting with the geoapi outside of the 2 clients would make it Guest-Unknown.

Reply:
That makes sense. Thanks!